app/src/main/app/sql/initMdi.sql

```diff
@@ -11,5 +13,5 @@ exec xpcasInitMdi 'ColumnSecurity', 'ColumnSecurity', '@xtcasColumnSecurity', 'C
 exec xpcasInitMdi 'UserGroup', 'UserGroup', '@xtcasUserGroup', 'CAS', 12.0, 'admin', 1, 'aero.minova.cas'
 exec xpcasInitMdi 'UserPrivilege', 'UserPrivilege', '@xtcasUserPrivilege', 'CAS', 13.0, 'admin', 1, 'aero.minova.cas'
 exec xpcasInitMdi 'Mdi', 'Menu', '@xtcasMdi', 'CAS', 14.0, 'admin', 1, 'aero.minova.cas'
```

Comment:
Der JPA-Code soll in Zukunft den SQL-Code ersetzen, oder?
Dann würde ich in dem SQL-Code zumindestns ein solches TODO hinzufügen.

Alternativ ein Issue im CAS dafür erstellen.

Reply:
Komplett ersetzten glaube ich nicht. Für "normale" 12er Projekte z.B. ohne Extension wäre es ärgerlich, nur für die Menü-Definition eine Extension schreiben zu müssen.

Oder meinst du nur das init-Mdi vom CAS? 
Das könnte im Prinzip jetzt schon entfernt werden, der einzige Vorteil den ich sehe ist als Vorlage für andere Projekte.

Reply:
Habe ein Issue erstellt, #838